### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by faizal101**
A few small edits

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,32 +24,32 @@ Articles are written in [DocFx-flavored Markdown](http://dotnet.github.io/docfx/
...
+For each Markdown file, there may be a folder for images and a folder for sample code. For example, if the article is [fundamentals/configuration/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/configuration/index.md), the images are in [fundamentals/configuration/index/\_static](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/configuration/index/_static) and the sample application project files are in [fundamentals/configuration/index/sample](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/configuration/index/sample). An image in the *fundamentals/configuration/index.md* file is rendered by the following Markdown:
```
[the remark this answers is not in the document]
Comment (faizal101) [reply CONTRIBUTING.md]: Could I know your reasoning on why "app" would be better as opposed to "application"?

**Comment by faizal101** [comment]
Thanks for your review! I believe almost all your suggestions sounds reasonable and i'm happy with it, apart from one part which I commented on.
Also, I prefer if you could change the PR yourself, mostly because there's a good chance i'll forget about it in the morning :P

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (guardrex) [reply CONTRIBUTING.md]: It's a repo rule. We use "app" almost everywhere.

**Comment by guardrex** [comment]
Sure ... will do. Thanks again for your help.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (faizal101) [reply CONTRIBUTING.md]: Alright, so that's why "app" should be used instead. Thanks!

**Comment by scottaddie** [comment]
Thank you for your help, @faizal101!

**Pull request merged**